### DISCUSSION TITLE
can audacity run in ubuntu docker, without GUI and sound card, cmd way to start

### DISCUSSION THREAD

**Discussion by didadida-r** [audacity/audacity#7619] [Development]
### Bug description

hi, how can we run audacity in ubuntu server, thanks.

### Steps to reproduce

run in linux,

### Expected behavior

successfully run in linux, without GUI

### Audacity Version

Audacity 3.7.x

### Operating system

Linux

**Comment by LWinterberg** [comment]
That's not something we support. Try `sox` or `ffmpeg` instead.

**Reply by didadida-r** [reply]
Thanks
